tests/jobs/test_scheduler_job.py

```diff
@@ -3641,35 +3640,35 @@ def test_execute_queries_count_no_harvested_dags(self, expected_query_count, dag
             # pylint: disable=bad-whitespace
             # expected, dag_count, task_count, start_ago, schedule_interval, shape
             # One DAG with one task per DAG file
-            ([ 8,   8,   8,  8],  1,  1, "1d",  "None",  "no_structure"),  # noqa
-            ([ 8,   8,   8,  8],  1,  1, "1d",  "None",        "linear"),  # noqa
-            ([20,  12,  12, 12],  1,  1, "1d", "@once",  "no_structure"),  # noqa
-            ([20,  12,  12, 12],  1,  1, "1d", "@once",        "linear"),  # noqa
-            ([20,  22,  25, 28],  1,  1, "1d",   "30m",  "no_structure"),  # noqa
-            ([20,  22,  25, 28],  1,  1, "1d",   "30m",        "linear"),  # noqa
-            ([20,  22,  25, 28],  1,  1, "1d",   "30m",   "binary_tree"),  # noqa
-            ([20,  22,  25, 28],  1,  1, "1d",   "30m",          "star"),  # noqa
-            ([20,  22,  25, 28],  1,  1, "1d",   "30m",          "grid"),  # noqa
+            ([10,  10,  10, 10],  1,  1, "1d",  "None",  "no_structure"),  # noqa
```

Comment:
How come the query counts changed? Oh, this is writing to the S10nDag table(s) right?

Reply:
This test calls `job._do_scheduling` -> `_change_state_for_tis_without_dagrun` (2x) -> Updates Task Instance table.

So every time we call `job._do_scheduling` we will have these 2 queries